Tools/TestWebKitAPI/Tests/IPC/ConnectionTests.cpp

```diff
@@ -1081,8 +1081,12 @@ TEST_P(ConnectionRunLoopTest, SendLocalSyncMessageWithDataReply)
         auto sendResult = a()->sendSync(MockTestSyncMessageWithDataReply { }, i, IPC::Timeout::infinity());
         ASSERT_TRUE(sendResult.succeeded());
         auto& [replyData] = sendResult.reply();
-        for (size_t i = 0; i < replyData.size(); ++i)
-            ASSERT_EQ(static_cast<uint8_t>(i), replyData[i]);
+        for (size_t i = 0; i < replyData.size(); ++i) {
+            auto expected = static_cast<uint8_t>(i);
+            if (expected != replyData[i])
+                ASSERT_EQ(expected, replyData[i]);
```

Comment:
So `ASSERT_EQ` will spend time constructing the assertion failure error message or something?

Reply:
I think GTest records the asserts that it ran and is able to list those. 
Or it's just that the implementation is slowish.